gretty 3.0.2 java.net.ConnectException: Connection refused

I have already tried version 3.0.2 but there's error about connection refused. is there any new config?
Edit1: I am using gradle 6.3-rc1
Edit2: I already tried gradle 6.3 but still error

```
Caused by: java.net.ConnectException: Connection refused: connect
        at org.akhikhl.gretty.ServiceProtocol$Writer.write(ServiceProtocol.groovy:76)
        at org.akhikhl.gretty.ServiceProtocol$Writer$write.call(Unknown Source)
        at org.akhikhl.gretty.LauncherBase$_beforeLaunch_closure2.doCall(LauncherBase.groovy:71)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.akhikhl.gretty.LauncherBase.beforeLaunch(LauncherBase.groovy:69)
        at org.akhikhl.gretty.DefaultLauncher.super$2$beforeLaunch(DefaultLauncher.groovy)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.akhikhl.gretty.DefaultLauncher.beforeLaunch(DefaultLauncher.groovy:58)
        at org.akhikhl.gretty.LauncherBase.launch(LauncherBase.groovy:155)
        at org.akhikhl.gretty.Launcher$launch.call(Unknown Source)
        at org.akhikhl.gretty.StartBaseTask.action(StartBaseTask.groovy:79)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
```

I can reproduce using one of the integration test apps in this repo, `helloJersey`:
```
me@mbp:helloJersey$ git checkout v3.0.2
me@mbp:helloJersey$ gradle appRun

> Configure project :farmSecure:MyWebApp
Excluding farmSecure tests from Jetty 9.3/9.4, see https://github.com/gretty-gradle-plugin/gretty/issues/67 .
Excluding farmSecure tests from Jetty 9.3/9.4, see https://github.com/gretty-gradle-plugin/gretty/issues/67 .

> Task :helloJersey:appRun FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':helloJersey:appRun'.
> java.net.BindException: Can't assign requested address (Bind failed)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 43s
4 actionable tasks: 4 executed
me@mbp:helloJersey$ gradle --version

------------------------------------------------------------
Gradle 5.4.1
------------------------------------------------------------

Build time:   2019-04-26 08:14:42 UTC
Revision:     261d171646b36a6a28d5a19a69676cd098a4c19d

Kotlin:       1.3.21
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          1.8.0_192 (Oracle Corporation 25.192-b12)
OS:           Mac OS X 10.13.6 x86_64
```
That same command works fine when using 3.0.1.

As far as I know, the app where I originally observed this is not using `gretty_ports.properties`. The gretty parts of its `build.gradle` look something like this:
```
buildscript {
    repositories {
        jcenter()
    }

    dependencies {
        classpath 'org.gretty:gretty:3.0.1'
    }
}

apply plugin: 'org.gretty'

gretty {
    extraResourceBase "webapp"
    extraResourceBase "internal"

    httpPort = 9100
    host = 'localhost' 
    contextPath = "/app"
    fastReload = true
    jvmArgs = [
        "-Dlog4j.configuration=${projectDir.toString()}/dev.logging.lcf"
    ]
    debugSuspend = true
    jacocoEnabled false
    servletContainer = 'jetty9.4'
}
```

--- a/libs/gretty-core/src/main/groovy/org/akhikhl/gretty/LauncherBase.groovy
+++ b/libs/gretty-core/src/main/groovy/org/akhikhl/gretty/LauncherBase.groovy
@@ -59,20 +59,6 @@ abstract class LauncherBase implements Launcher {
 
     File portPropertiesFile = getPortPropertiesFile()
     portPropertiesFile.parentFile.mkdirs()
-    if(portPropertiesFile.exists()) {
-      Properties portProps = new Properties()
-      portPropertiesFile.withReader 'UTF-8', {
-        portProps.load(it)
-      }
-      int servicePort = portProps.servicePort as int
-      int statusPort = portProps.statusPort as int
-      ServiceProtocol.createReader(statusPort).withCloseable { reader ->
-        def response = reader.readMessageAsync()
-        ServiceProtocol.createWriter(servicePort).write('status')
-        if (response.get() == 'started')
-          throw new RuntimeException('Web-server is already running.')
-      }
-    }
   }
 
   @Override

